packages/react-composites/src/composites/CallComposite/components/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '@internal/react-components/src/components/DevicesButton';
+
+/**
+ * @internal
+ */
+export interface LocalVideoCameraButtonProps {
+  /** Array of cameras available to the user. */
+  cameras: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  currentCamera: OptionsDevice;
+  /** callback function to change video feed. */
+  setCamera: (device: OptionsDevice) => Promise<void>;
+}
+
+const localVideoCameraCycleButtonStyles: IButtonStyles = {
+  root: {
+    position: 'absolute',
+    right: '0',
+    color: 'white', // only shows up on running video feed to we want to force specific colours.
+    zIndex: 4
```

Comment:
Try give clarity why this is zIndex `4` - why 4? what is it trying to be above/below?
If reasonable use +1 of whatever it should be above similar to:
![image](https://user-images.githubusercontent.com/2684369/151258862-e25e8d28-51e5-4e81-9c1f-a855a3193501.png)

Reply:
oops just saw this, will include fix to this comment in the PR for the logic